pages/index.tsx

```diff
@@ -2,11 +2,18 @@ import { GetStaticProps } from 'next';
 import { handleImages } from 'src/utils/imageHandler';
 import { Employee, massageEmployee } from 'src/employees';
 import { getEmployeesUrl } from 'src/utils/api/getEmployees';
+import { CaseJSON } from '../src/case/Case';
+import { CaseList } from '../src/case/cases';
 
 export { default } from 'src/index';
 
+function shuffle(array: Array<CaseJSON>) {
```

Comment:
Here you only use it for one thing here, so not necessary, but I think this is a perfect use case for [generics](https://www.typescriptlang.org/docs/handbook/generics.html). Example:

```
function shuffle<T>(array: Array<T>) { }
```

Reply:
good call, going over it now :D